Fixes the inconsistency between the text and code blocks (#1195)
wandb/docs: content/guides/models/sweeps/walkthrough.md

> The proceeding example demonstrates a sweep configuration that uses a random search (`'method':'random'`). The sweep will randomly select a random set of values listed in the configuration for the batch size, epoch, and the learning rate.

Reviewer: @johndmulhausen This is an interesting one....do we want GPT Editor to suggest using the product name (common noun) as a verb?

> The proceeding example demonstrates a sweep configuration that uses a random search (`'method':'random'`). The sweep will randomly select a random set of values listed in the configuration for the batch size, epoch, and the learning rate.

Reviewer 2: It looks like this got misaligned; this suggestion applies to line 80.

> The proceeding example demonstrates a sweep configuration that uses a random search (`'method':'random'`). The sweep will randomly select a random set of values listed in the configuration for the batch size, epoch, and the learning rate.

Reviewer 2: And no, using "sweep" as a verb is not okay

> Throughout the sweeps, W&B will maximize the metric specified in the metric key (`metric`). In the following example, W&B will minimize (`"goal": "minimize"`) the score (`"name": "score"`).

Reviewer 2: Misaligned suggestion. Closing.

> Throughout the sweeps, W&B will maximize the metric specified in the metric key (`metric`). In the following example, W&B will minimize (`"goal": "minimize"`) the score (`"name": "score"`).

Reviewer 2: ```suggestion
W&B minimizes the metric specified in the `metric` key when `"goal": "minimize"` is associated with it. In this case, W&B will optimize for minimizing the metric  `score` (`"name": "score"`).
```

> 

Reviewer 2: Misaligned suggestion (Would add edited line to line 86 instead of revising line 84)

> ## Stop the agent (optional)

Reviewer 2: We definitely want to use the term "agent" and not "assistant"

## Stop the agent (optional)

> In the terminal, press `Ctrl+C` to stop the current run. Press it again to stop the agent entirely.

Reviewer 2: ```suggestion
In the terminal, press `Ctrl+C` to stop the current run. Press it again to terminate the agent.
```
